src/App.js

```diff
@@ -1,24 +1,74 @@
+import React,{useState,useEffect} from 'react';
 import './App.css';
 
+import Container from './components/container';
+import ItemListContainer from './components/ItemListContainer';
+import TodoFormContainer from './components/TodoFormContainer'
+
+import uuidv4 from './utils/random';
+
 function App() {
+
+  const [todoList,setTodoList] = useState([]);
+
+  useEffect(() => {
+    loadDataFromLocalStorage()
+  },[]);
+
+  useEffect(()=>{
+    const data = todoList;
+    saveToLocalStorage(data);
+  },[todoList])
+
+  // todoList의 isDone state를 변경
+  const handleList = (data) => {
+    const found = todoList.findIndex(element=>element.id===data.id);
+    let myarr = [...todoList];
+    myarr[found]=data;
+    setTodoList(myarr);
+  }
+  
+  // todoList 삭제
+  const handleDelete = (id) => {
+    const newList = todoList.filter(element=>element.id!==id);
+    setTodoList(newList);
+  }
+
+  // form에서 입력받은대로 submit결과를 핸들링
+  const handleSubmit = (input) => {
+    if(input !== undefined && input !== ''){
```

Comment:
```suggestion
    if(input){
```

`undefined`, `null` 모두 js에서는 falsy기 때문에 이렇게 적을수 있습니다! [falsy란?](https://developer.mozilla.org/ko/docs/Glossary/Falsy)

Reply:
falsy에 대해 공부하는 시간을 가졌습니다! '' 이것 또한 falsy한 값인건 처음 알았습니다. javascript는 정말 신기한 친구네요..